torch/_dynamo/variables/misc.py

```diff
@@ -790,20 +790,34 @@ def call_function(
             return variables.ListIteratorVariable(
                 items, mutable_local=MutableLocal(), **options
             )
-        elif self.value is itertools.accumulate and not kwargs:
+        elif self.value is itertools.accumulate:
             from .builtin import BuiltinVariable
 
+            if any(key not in ["initial", "func"] for key in kwargs.keys()):
+                raise unimplemented("Unsupported kwarg for itertools.accumulate")
```

Comment:
Nit: print the unsupported kwarg name in message

Reply:
Also you don't raise unimplemented, just call it. (There is another site below that makes the same mistake)

Reply:
fixed